packages/react-strict-animated/src/native/components.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ */
+
+/* eslint-disable no-unreachable */
+
+import type { AnimatedStyleValue } from '../shared/SharedAnimatedTypes';
+import type { Text, View } from 'react-native';
+import type { ImageProps } from 'react-native/Libraries/Image/ImageProps';
+import type { html } from 'react-strict-dom';
+
+import * as React from 'react';
+import { Animated } from 'react-native';
+
+import { compat, css } from 'react-strict-dom';
+
+const styles = css.create({
+  defaults: {
+    boxSizing: 'content-box',
+    position: 'static'
+  }
+});
+
+export component AnimatedDiv(
+  animatedStyle?: AnimatedStyleValue<Animated.Node>,
+  children?: React.Node,
+  ref?: React.RefSetter<React.RefOf<html.div>>,
+  ...htmlProps: Omit<React.PropsOf<html.div>, 'children'>
+) {
+  return (
+    // $FlowFixMe[prop-missing] - RSD missing ref type on compat.native API
+    <compat.native
+      {...htmlProps}
+      as="div"
+      ref={ref}
+      style={[styles.defaults, htmlProps.style]}
+    >
+      {(nativeProps: React.PropsOf<View>) => {
+        return (
+          <Animated.View
+            {...nativeProps}
+            style={[nativeProps.style, animatedStyle]}
```

Comment:
Can `animatedStyle` be passed to `compat.native` so it picks up any relevant polyfills?